src/main/java/info/fingo/urlopia/request/Request.java

```diff
@@ -87,7 +97,83 @@ public LocalDate getEndDate() {
         return endDate;
     }
 
-    public String getMailContent() {
-        return mailContent;
+    public Type getType() {
+        return type;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public TypeInfo getTypeInfo() {
+        TypeInfo typeInfo = null;
+
+        TypeInfo typesInfo[] = OccasionalType.values();
+        for(TypeInfo tempTypeInfo : typesInfo) {
+            if(tempTypeInfo.getName().equals(this.typeInfo)) {
+                typeInfo = tempTypeInfo;
+                break;
+            }
+        }
+
+        return typeInfo;
+    }
+
+    public void setTypeInfo(TypeInfo typeInfo) {
+        this.typeInfo = typeInfo.getName();
+    }
+
+    public Status getStatus() {
+        return status;
+    }
+
+    public void setStatus(Status status) {
+        this.status = status;
+    }
+
+    public enum Type {
+        NORMAL,
+        OCCASIONAL
+    }
+
+    public interface TypeInfo {
+        String getInfo();
+        String getName();
+    }
+
+    public enum OccasionalType implements TypeInfo {
+        WRONG (0, "Niepoprawny typ okazjonalny"),
+        D2_BIRTH (1, "Narodziny dziecka"),
+        D2_FUNERAL (2, "Zgon/pogrzeb osoby z najbliższej rodziny"),
+        D2_WEDDING (3, "Ślub"),
+        D1_FUNERAL (4, "Zgon/pogrzeb osoby bliskiej"),
+        D1_WEDDING (5, "Ślub dziecka");
```

Comment:
wiem, że być powinny, ale w obecnej sytuacji trochę czas gonił i nie chciało mi się rozkminiać jak to porządnie zrobić - zrobię przy okazji bo jakieś locale na backu też by się przydały... chyba, że podam tu klucze, a wtedy front zajmie się tłumaczeniem :) To jest myśl... ale chyba nie na dzisiaj :P